configs/components/cmake.rb

```diff
@@ -1,12 +1,14 @@
 component "cmake" do |pkg, settings, platform|
   # Source-Related Metadata
-  pkg.version "3.2.3"
-  pkg.md5sum "d51c92bf66b1e9d4fe2b7aaedd51377c"
   if platform.name =~ /fedora-f24/
```

Comment:
Since we don't support fedora 24 anymore, can we just take out this conditional?

Reply:
@mwaggett we will remove it outside this PR, but yes.